qiita_pet/handlers/qiita_redbiom.py

```diff
@@ -43,13 +41,17 @@ def _redbiom_metadata_search(self, query, contexts):
                 'check the search help for more information on the queries.'
                 % query)
         if not message:
-            study_samples = defaultdict(list)
-            for s in samples:
-                study_samples[s.split('.', 1)[0]].append(s)
-            for sid, samps in viewitems(study_samples):
-                study_artifacts[sid] = {
-                    a.id: samps for a in Study(sid).artifacts(
-                        artifact_type='BIOM')}
+            study_artifacts = defaultdict(lambda: defaultdict(list))
+            for ctx in contexts:
+                # redbiom.fetch.data_from_samples returns a biom, which we
+                # will ignore, and a dict
+                _, data = redbiom.fetch.data_from_samples(ctx, redbiom_samples)
+                for vals in data.values():
+                    for idx in vals:
+                        print idx
```

Comment:
still want that print statement in there?

Reply:
left by mistake, one second!